pkg/azuredisk/azuredisk_v2.go

```diff
@@ -423,13 +424,16 @@ func (d *DriverV2) StartControllersAndDieOnExit(ctx context.Context) {
 		defer func() { w.Finish(errors...) }()
 		// recover lost states if necessary
 		w.Logger().Infof("Elected as leader; initiating CRI deperecation / recovery...")
-		if err := nodeReconciler.Recover(ctx); err != nil {
+		// Use unique recoveryID as the Recover Annotation, so if it recovers more than once(for example: driver restarts or upgrades)
+		// the objects can still get a new Recover Annotation to force register an UPDATE event for reconciliation.
+		recoveryID := uuid.NewUUID().String()
```

Comment:
It would be good to include the recovery ID in the log trace above so that we can match the recovered CRIs to the driver logs.

Reply:
Done.